File: python_ruby workspace/class_member.py
class Cs :
    @staticmethod
    #클래스에 직접 포함된 메소드를 의미함
    def static_method() :
        print("static method")
    @classmethod
    #클래스에 직접 포함된 메소드를 의미함
    def class_method(cls) : 
        print("Class method")
    def instance_method(self):
        print("instance method")

i = Cs()
Cs.static_method()
Cs.class_method()
i.instance_method()

File: python_ruby workspace/class_member.rb
class Cs
    def Cs.class_method() #클래스에 직접 포함된 메소드를 의미함
        p "Class method"
    end
    def instance_method()
        p "instance_method"
    end
end

i = Cs.new()
Cs.class_method()
i.instance_method()
# Cs.instance_method()
i.class_method()